features/in_onderzoek.feature

```diff
@@ -666,24 +783,23 @@ Functionaliteit: in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550001 |
-      En de persoon heeft volgende 'overlijden' gegevens
-      | naam  | waarde   |
-      | datum | 20220218 |
-      En de persoon heeft volgende 'overlijden.inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      En de persoon heeft de volgende 'overlijden' gegevens
+      | naam                                     | waarde   |
+      | datum                                    | 20220218 |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde> |
+      | datum ingang onderzoek (83.20)           | 20220307 |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | overlijden.indicatieOverleden   |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'overlijden.inOnderzoek' gegevens
-      | naam                             | waarde       |
-      | indicatieOverleden               | true         |
-      | datumIngangOnderzoek.datum       | 2022-03-07   |
-      | datumIngangOnderzoek.type        | Datum        |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+      | naam                                         | waarde       |
+      | indicatieOverleden                           | true         |
+      | inOnderzoek.indicatieOverleden               | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
 
       Voorbeelden:
       | reden                | waarde |
```

Comment:
Wijzig in

| gegeven in onderzoek | waarde |

Reply:
aangepast